### PR TITLE
Eeitt content changes

### PR DESCRIPTION
**Pull request by HarrisonBaxter**

### CODE REVIEW

**Comment by enriquefaci** [comment]
Hi,

Those changes looks fine. However, I would advise having a look at this document:
https://github.com/hmrc/hmrc-email-renderer/blob/master/CONTRIBUTING.md

We provide some content/design guidelines about email templates.

In you template a tittle is missing and there is not any salutation at all. We provide a helper for the salutation. if you don't provide any name (as parameter) it will say Dear Customer which is not great but it is probably better than saying nothing.

**Comment by HarrisonBaxter** [comment]
Hi Enrique, 

I'll pass this on to our content designers and get them to come up with something else

**Comment by HarrisonBaxter** [comment]
I have changed this template to have a test title as  it's not going to be for public use until much later

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/EeittTemplates.scala [hunk 1]
```diff
@@ -27,7 +27,7 @@ object EeittTemplates {
       templateId = "eeitt_submission_confirmation",
       fromAddress = govUkTeamAddress,
       service = Eeitt,
-      subject = "Thank you for your submission",
+      subject = "Confirmation",
```
Comment (drainhmrc) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/EeittTemplates.scala]: Change from Confirmation to "Submission Confirmation"

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/eeittSubmissionConfirmation.scala.html [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
\ No newline at end of file
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "This is a test email, should you receive it please ignore it"){
+<p style="margin: 0 0 30px; font-size: 19px;">
+    Dear Customer <br>
+    This is a confirmation of your submission to <br>HMRC.<br>
+    <br>
+    Thank you.<br>
+    <br>
+    HMRC submissions team<br>
+    <br>
```
Comment (drainhmrc) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/eeittSubmissionConfirmation.scala.html]: remove do not reply to this email

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/eeittSubmissionConfirmation.scala.txt [hunk 3]
```diff
@@ -1,9 +1,13 @@
-@(params: Map[String, Any]) Thanks for the submission. This is a test email, should you receive it ignore it
-Submission date: @{params("submissionDate")}
+@(params: Map[String, Any]) This is a test email, should you receive it please ignore it
```
Comment (drainhmrc) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/eeittSubmissionConfirmation.scala.txt]: Remove space before " This"

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/eeittSubmissionConfirmation.scala.txt [hunk 4]
```diff
@@ -1,9 +1,13 @@
...
 
-You should contact: @{params("contactInfo")}
+This is a confirmation of your submission to
+HMRC.
 
-You can call @{params("telephoneNumber")} for help
+Thank you.
+
+HMRC submissions team
+
```
Comment (drainhmrc) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/eeittSubmissionConfirmation.scala.txt]: Remove code below this.

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/eeittSubmissionConfirmation.scala.txt [hunk 5]
```diff
@@ -1,9 +1,13 @@
-@(params: Map[String, Any]) Thanks for the submission. This is a test email, should you receive it ignore it
-Submission date: @{params("submissionDate")}
+@(params: Map[String, Any]) This is a test email, should you receive it please ignore it
```
Comment (drainhmrc) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/eeittSubmissionConfirmation.scala.txt]: Add Extra new line

**Pull request merged**